### ISSUE TITLE
[css-values] Should round(strategy, A, 0) be NaN or A?

### ISSUE THREAD

**Issue by Loirooriol** [w3c/csswg-drafts#4762]
Currently https://drafts.csswg.org/css-values-4/#round-infinities says

> In `round(A, B)`, if B is 0, the result is NaN.

But according to @tabatkins "my aesthetic sense is telling me I'm not allowed to say it's NaN when there are clear limit values."

In this case the limit is clear: `lim_{x→0} round(strategy, A, x) = A`.
So maybe `round(strategy, A, 0)` should be `A`?

On the other hand, being `A` would seem to imply that `A` is a multiple of 0, which technically it's false.
But `A / ∞ = 0`, so in a sense we do have `A ≈ 0 * ∞`, dunno.

**Comment by Crissov** [comment]
`round(A, zero)` should intuitively equal `round(A)` and thus `A` indeed.

**Comment by Loirooriol** [comment]
@Crissov Note the spec doesn't allow `round(A)`, and if it did I would expect it to be `round(A, 1)` (i.e. round to integer), not `round(A, 0)`.

I don't have a strong opinion about `round(A, 0)` being `A` or NaN, but the choice doesn't seem obvious, so I filed this so that people can give some thought to it.

**Comment by tabatkins** [comment]
Yeah, in this case the limit is different from the behavior *at* the limit, I think. The invariant that the result is a multiple of the step is broken.

> But `A / ∞ = 0`, so in a sense we do have `A ≈ 0 * ∞`, dunno.

Note that `0 * infinity` is NaN. ^_^

**Comment by Loirooriol** [comment]
Sure, that's why I said "in a sense", i.e. it's one of the possible solutions of the indetermination, but there are others.
